django/contrib/admin/static/admin/css/forms.css

```diff
@@ -37,6 +37,7 @@ form .form-row p {
 /* FORM LABELS */
 
 label {
+    order: -1;
```

Comment:
Although this does work, I feel this might not be the best approach

I feel the issue is coming from (see 999ba9db6d6331eaa58af77debba42754bcc1a8e):
https://github.com/django/django/blob/0d1dd6bba0c18b7feb6caa5cbd8df80fbac54afd/django/contrib/admin/static/admin/js/SelectFilter2.js#L36 
When we remove this line, the label is before the div and it renders correctly.

I think we should see if we can have the html (generated from javascript) be in the correct order rather than adding CSS rules to "fix" the incorrect order

Reply:
Thank you so much for the review @sarahboyce ! I will hack around that javascript script code and see if I can figure this out. TBH I could not figure out the exact location of where these elements were being created, so I appreciate your hint.

Reply:
QQ: I suppose that I will need to alter/add a javascript test. Do you think that the screenshot tests are still appropriate here?

Reply:
> QQ: I suppose that I will need to alter/add a javascript test. Do you think that the screenshot tests are still appropriate here?

I'm not sure. We might end up removing them :thinking: